src/hex/shape.rs

```diff
@@ -21,34 +21,12 @@ use serde::{Deserialize, Serialize};
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[derive(Clone, PartialEq, Debug, Default)]
 pub struct HexShape<T: Clone> {
-    shape: Array2<Option<T>>, // 2D array of tiles, index corresponds to coordinate (q, r). Origin of (0,0) to (+∞,+∞).
-    pub transform: Transform<Axial>, // The transformation matrix to convert from parent grid to local space.
+    /// 2D array of tiles, index corresponds to coordinate (q, r). Origin of (0,0) to (+∞,+∞).
+    shape: Array2<Option<T>>,
+    /// The transformation matrix to convert from parent grid to local space.
+    pub transform: Transform<Axial>,
 }
 
-/// Struct to handle arguments to shape constructors.
-///
-/// `size` field denotes length.
-///
-/// `rot_dir`: positive denotes CW, negative CCW, magnitude denotes how many 60 degree rotations.
-pub struct ShapeArgs {
```

Comment:
Dead code again, I assume?

Reply:
Yea more dead code since I removed the struct usage, if anything they'll be tuples in the future, but Clippy suggests shortening parameters once you get to 7/9 long actually so even that number should be fine.